test/AutobahnTestApp/AutobahnTestApp.csproj

```diff
@@ -1,16 +1,20 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  
```

Comment:
I didn't complain about your style when creating new files. But changing the style when making real changes just makes reviewers' jobs harder.